app/models/tenant.rb

```diff
@@ -88,6 +88,15 @@ def disable_feature(feature)
     save!
   end
 
+  def list_features
+    AVAILABLE_FEATURE_FLAGS.map do |feature|
+      { name: feature.to_s,
+        enabled: feature_enabled?(feature),
+        features_including: (feature_flags + [feature.to_s]).join(","),
```

Comment:
Toto je interna vec view - cize tam by som to takto rozdeloval, plus teda hladas `partition` metodu.

Reply:
@luciajanikova vies tu napisat flags ktore nema byt vidiet - resp. len v ?labs

Reply:
Viditelne flagy: `:audit_log`, `:archive`, `:api`, `:fs_sync`
Schovane flagy: `:message_draft_import`, `:fs_api`
podla mna takto @jsuchal

Reply:
@richardlences vies takto nastavit?

Reply:
Ano nastavil som to tak